sechub-pds-solutions/multi/docker/database/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: MIT
+
+FROM ubuntu:20.04
+
+ENV POSTGRE_CONFIG_FOLDER="/etc/postgresql/12/main/"
```

Comment:
Is it intentionally named "POSTGRE_CONFIG_FOLDER"?
For me "POSTGRES_CONFIG_FOLDER" would be much better readable.